imports/pages/groups/profile/index.js

```diff
@@ -60,16 +103,44 @@ class TemplateWithoutData extends Component {
     this.props.dispatch(modal.hide());
   }
 
-  sendRequest = (e, callback) => {
+  onPhoneNumberChange = (value: string) => {
+    const phoneNumber = value.replace(/[^\d]+/g, "");
+    return this.setState({ phoneNumber });
+  }
+
+  validatePhoneNumber = (value: string): boolean => {
```

Comment:
@JakeDawkins I agree that it would be great to move this out later. Not so necessary right now.